modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/param/converter/provider/SiteParamConverterProvider.java

```diff
@@ -46,27 +48,17 @@ public Long fromString(String parameter) {
 		MultivaluedMap<String, String> multivaluedMap =
 			_uriInfo.getPathParameters();
 
-		Long siteId = null;
-
-		if (multivaluedMap.containsKey("assetLibraryId")) {
-			siteId = getDepotGroupId(
-				multivaluedMap.getFirst("assetLibraryId"),
-				_company.getCompanyId());
-		}
-		else {
-			siteId = getGroupId(
-				_company.getCompanyId(), multivaluedMap.getFirst("siteId"));
-		}
+		Long groupId = _getGroupId(multivaluedMap, parameter);
```

Comment:
I assume that the name change (siteId to groupId) is made with a reason

Reply:
siteId was not really correct, as it could be the ID of a site or an asset library. "groupId" is a good enough middle ground.

Reply:
See https://github.com/liferay-content-management/liferay-portal/pull/6508/commits/9b248e2687c6795c746055e7f993787802d73218